src/java/Light_vehicle.java

```diff
@@ -0,0 +1,36 @@
+package java;
+
+import java.util.Scanner;
+
+public class Light_vehicle extends Land_transport{
+    String TypeOfBody;
+    int NumberOfSeats;
+    private int travel;
+    private double a;
+    private int time;
+
+
+    @Override
+    void Vyvod() {
+        super.Vyvod();
+        System.out.println("Type of body: " + TypeOfBody);
+        System.out.println("Number of seats: " + NumberOfSeats);
+    }
+    private void rashod(){
+         a =travel/FuelConsumption;
+    }
+    void Dvizhenije(){
+        System.out.println("Enter travel time: ");
+        Scanner scanner = new Scanner(System.in);
+        time=scanner.nextInt();
+        travel=time*maksSpeed;
+        rashod();
+        System.out.println("During the time "+time+" "+"h "+" "+ "car "+" "+ carModel+" " + "moving with the maximum speed " +" "+maksSpeed+"km/h "+" "+
+                "will travel "+" "+travel + "km "+""+"and consume "+" "+ a +" " + " liters of fuel" );
+    }
```

Comment:
@Hlieb операторы и пробелы + убрать лишнюю конкатенацию строк.
```suggestion
        System.out.println("During the time " + time + " h, the car " + carModel + " moving with the maximum speed " + maksSpeed + " km/h " + "will ride " + travel + " km and consume " + a + " liters of fuel" );
    }
```